Makefile.am

```diff
@@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
 
 libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
 	-export-symbols-regex '^fjson_.*'
-libfastjson_la_CPPFLAGS = -Werror
```

Comment:
I need to correct myself. AC_CHECK_FUNCS is used and of course also affected. So we need to have a working solution before I can remove. give me a day or two to finish some related work, I think we might otherwise do some changes that get dropped almost immediately.